File: cpp/coordenadas.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    double x, y;

    cout << "Valor de X: ";
    cin >> x;

    cout << "Valor de Y: ";
    cin >> y;


    if(x == 0 && y == 0){
        cout << "Origem" << endl;
    }
    else if(x == 0){
        cout << "Eixo Y" << endl;
    }
    else if(y == 0){
        cout << "Eixo X" << endl;
    }
    else if(x > 0 && y > 0){
        cout << "Q1" << endl;
    }
    else if(x < 0 && y > 0){
        cout << "Q2" << endl;
    }
    else if(x < 0 && y < 0){
        cout << "Q3" << endl;
    }
    else{
        cout << "Q4" << endl;
    }

	return 0;
}

File: python/coordenadas.py
x: float; y: float

x = float(input("Valor de X: "))
y = float(input("Valor de Y: "))

if x == 0 and y == 0:
	print("Origem")
elif x == 0:
	print("Eixo Y")
elif y == 0:
	print("Eixo X")
elif x > 0 and y > 0:
	print("Q1")
elif x < 0 and y > 0:
	print("Q2")
elif x < 0 and y < 0:
	print("Q3")
else:
	print("Q4")
